.travis.yml

```diff
@@ -1,14 +1,21 @@
 language: python
-python: 2.7
+
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.8-dev"
```

Comment:
I would keep `2.7` too, however, up to you. Now flake8 is passed on both Python versions (verified locally).

Reply:
Done, added 2.7.